tables/TaQL/ExprNode.cc

```diff
@@ -1494,20 +1494,17 @@ void TableExprNode::adaptUnit (const Unit& unit)
 TableExprNode TableExprNode::newRownrNode (const TableExprInfo& tableInfo,
                                            uInt origin)
 {
-    TENShPtr tsnptr = new TableExprNodeRownr (tableInfo, origin);
-    return tsnptr;
+    return TENShPtr(new TableExprNodeRownr (tableInfo, origin));
```

Comment:
Optional:
```suggestion
    return std::make_shared<TableExprNodeRownr>(tableInfo, origin);
```
Similar improvements apply to the return statements in the functions below.

Reply:
done